[2025-07-12T14:18:42.916+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: flight_data_etl.extract_flight_api_data manual__2025-07-12T14:18:41+00:00 [queued]>
[2025-07-12T14:18:42.919+0000] {taskinstance.py:1956} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: flight_data_etl.extract_flight_api_data manual__2025-07-12T14:18:41+00:00 [queued]>
[2025-07-12T14:18:42.919+0000] {taskinstance.py:2170} INFO - Starting attempt 1 of 2
[2025-07-12T14:18:42.925+0000] {taskinstance.py:2191} INFO - Executing <Task(PythonOperator): extract_flight_api_data> on 2025-07-12 14:18:41+00:00
[2025-07-12T14:18:42.928+0000] {standard_task_runner.py:60} INFO - Started process 234 to run task
[2025-07-12T14:18:42.932+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'flight_data_etl', 'extract_flight_api_data', 'manual__2025-07-12T14:18:41+00:00', '--job-id', '36', '--raw', '--subdir', 'DAGS_FOLDER/flight_data_etl.py', '--cfg-path', '/tmp/tmprk5xvx0m']
[2025-07-12T14:18:42.937+0000] {standard_task_runner.py:88} INFO - Job 36: Subtask extract_flight_api_data
[2025-07-12T14:18:42.962+0000] {task_command.py:423} INFO - Running <TaskInstance: flight_data_etl.extract_flight_api_data manual__2025-07-12T14:18:41+00:00 [running]> on host b78d8dd28f9c
[2025-07-12T14:18:43.139+0000] {taskinstance.py:2480} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='data_engineer' AIRFLOW_CTX_DAG_ID='flight_data_etl' AIRFLOW_CTX_TASK_ID='extract_flight_api_data' AIRFLOW_CTX_EXECUTION_DATE='2025-07-12T14:18:41+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-07-12T14:18:41+00:00'
[2025-07-12T14:18:44.493+0000] {logging_mixin.py:188} INFO - Extracted 11422 flight records from API
[2025-07-12T14:18:44.496+0000] {python.py:201} INFO - Done. Returned value was: /opt/***/data/flight_api_data.csv
[2025-07-12T14:18:44.503+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=flight_data_etl, task_id=extract_flight_api_data, execution_date=20250712T141841, start_date=20250712T141842, end_date=20250712T141844
[2025-07-12T14:18:44.518+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2025-07-12T14:18:44.529+0000] {taskinstance.py:3280} INFO - 1 downstream tasks scheduled from follow-on schedule check
